migrations/versions/dc3d31b16660_adding_spiceness.py

```diff
@@ -11,7 +11,7 @@
 
 # revision identifiers, used by Alembic.
 revision = "dc3d31b16660"
-down_revision = "59a2db502263"
+down_revision = "91cb3f06946c"
```

Comment:
nie będą się teraz gryźć migracje po drugim Twoim PRze?

Reply:
Będzie, ale jak ten zamknę to tam zmienię